protocol/bc/asset.go

```diff
@@ -10,14 +10,14 @@ import (
 
 // AssetID is the Hash256 of the issuance script for the asset and the
 // initial block of the chain where it appears.
-type AssetID [32]byte
+type AssetID Byte32
```

Comment:
(Sorry, that comment from last night was a little rushed and unclear.) The protobuf version of `AssetID` will be defined in terms of `Hash`, but for this interim change I prefer to keep the representation of `AssetID` unchanged.